splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -1573,15 +1576,18 @@
           "const": "number"
         },
         "range": {
-          "type": "array",
-          "items": {
-            "type": "number"
-          }
+          "$ref": "#/definitions/RangeValue"
         }
       },
       "required": ["type", "range"],
       "additionalProperties": false
     },
+    "RangeValue": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      }
+    },
```

Comment:
in python we have sth like that 
```
        range_v = options.get("range")

        if range_v is not None:
            min_v, max_v = range_v
            definition["validators"].append(
                {
                    "type": "number",
                    "range": range_v,
                    "errorMsg": f"{self['label']} must be between {min_v} and {max_v}",
                }
            )
```
so shouldn't the RangeValue be restricted to 2 values?

cuz right now i think user can put there any array with any numbers

solution 1 - make array max size 2 
```
    "RangeValue": {
      "type": "array",
      "items": {
        "type": "number"
      },
      "minItems": 2,
      "maxItems": 2
    },

```

solution 2 - use object
```
    "RangeValue": {
      "type": "object",
      "properties": {
            "min_v": {
              "type": "number"
            },
            "max_v": {
              "type": "number"
            },
      },
      required: [min_v, max_v]
    },
```

Reply:
At first I used minItems and maxItems but this does not give a clear error message. However, I added this check to `global_config_validator.py`, similar to other range.